discord/types/guild.py

```diff
@@ -119,6 +119,7 @@ class Guild(_BaseGuildPreview, _GuildOptional):
     explicit_content_filter: ExplicitContentFilterLevel
     roles: List[Role]
     mfa_level: MFALevel
+    nsfw: bool
```

Comment:
Indent level on this seems off?

Reply:
These are four spaces, it is in line with all the other attributes. Did you mean the order of the attributes?

Reply:
I think my phone managed to mess this up, sorry. It looked off on the diff view but on desktop it looks fine.